2019-12-09 16:38:01,605 - root - INFO - Starting Process for store
2019-12-09 16:38:01,611 - root - INFO - Client channel established with the store
2019-12-09 16:41:02,124 - root - INFO - Starting Process for store
2019-12-09 16:41:02,126 - root - INFO - Client channel established with the store
2019-12-09 16:41:04,554 - root - INFO - Starting Process for store
2019-12-09 16:41:04,557 - root - INFO - Client channel established with the store
2019-12-09 16:41:06,989 - root - INFO - Starting Process for store
2019-12-09 16:41:06,991 - root - INFO - Client channel established with the store
2019-12-09 16:42:14,724 - root - INFO - Starting Process for store
2019-12-09 16:42:14,726 - root - INFO - Client channel established with the store
2019-12-09 16:53:13,140 - root - INFO - Starting Process for store
2019-12-09 16:53:13,143 - root - INFO - Client channel established with the store
2019-12-09 16:53:13,143 - root - WARNING - Map reduce started
2019-12-09 16:53:13,143 - root - WARNING - Dividing data in chunks- START
2019-12-09 16:53:48,034 - root - INFO - Starting Process for store
2019-12-09 16:53:48,037 - root - INFO - Client channel established with the store
2019-12-09 16:53:48,037 - root - WARNING - Map reduce started
2019-12-09 16:53:48,037 - root - WARNING - Dividing data in chunks- START
2019-12-09 16:57:21,632 - root - INFO - Starting Process for store
2019-12-09 16:57:21,636 - root - INFO - Client channel established with the store
2019-12-09 17:02:06,811 - root - INFO - Starting Process for store
2019-12-09 17:02:06,815 - root - INFO - Client channel established with the store
2019-12-09 17:02:06,815 - root - WARNING - Making RPC call to store
2019-12-09 17:05:05,011 - root - INFO - Starting Process for store
2019-12-09 17:05:05,014 - root - INFO - Client channel established with the store
2019-12-09 17:05:05,015 - root - WARNING - Making RPC call to store
2019-12-09 17:05:17,133 - root - INFO - Starting Process for store
2019-12-09 17:05:17,136 - root - INFO - Client channel established with the store
2019-12-09 17:05:17,137 - root - WARNING - Making RPC call to store
2019-12-09 17:05:46,110 - root - INFO - Starting Process for store
2019-12-09 17:05:46,113 - root - INFO - Client channel established with the store
2019-12-09 17:05:46,114 - root - WARNING - Making RPC call to store
2019-12-09 17:06:12,299 - root - INFO - Starting Process for store
2019-12-09 17:06:12,302 - root - INFO - Client channel established with the store
2019-12-09 17:06:12,303 - root - WARNING - Making RPC call to store
2019-12-09 17:07:31,696 - root - INFO - Starting Process for store
2019-12-09 17:07:31,705 - root - INFO - Client channel established with the store
2019-12-09 17:09:26,903 - root - INFO - Starting Process for store
2019-12-09 17:09:26,906 - root - INFO - Client channel established with the store
2019-12-09 17:09:26,906 - root - WARNING - Making RPC call to store
2019-12-09 17:14:09,479 - root - INFO - Starting Process for store
2019-12-09 17:14:09,482 - root - INFO - Client channel established with the store
2019-12-09 17:14:09,482 - root - WARNING - Making RPC call to store
2019-12-09 17:20:24,805 - root - INFO - Starting Process for store
2019-12-09 17:20:24,807 - root - INFO - Client channel established with the store
2019-12-09 17:20:24,808 - root - WARNING - Making RPC call to store
2019-12-09 17:21:02,254 - root - INFO - Starting Process for store
2019-12-09 17:21:02,257 - root - INFO - Client channel established with the store
2019-12-09 17:21:02,257 - root - WARNING - Map reduce started
2019-12-09 17:21:02,258 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:24:34,744 - root - INFO - Starting Process for store
2019-12-09 17:24:34,748 - root - INFO - Client channel established with the store
2019-12-09 17:24:34,748 - root - WARNING - Map reduce started
2019-12-09 17:24:34,748 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:24:46,106 - root - INFO - Starting Process for store
2019-12-09 17:24:46,109 - root - INFO - Client channel established with the store
2019-12-09 17:24:46,110 - root - WARNING - Map reduce started
2019-12-09 17:24:46,111 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:26:42,376 - root - INFO - Starting Process for store
2019-12-09 17:26:42,379 - root - INFO - Client channel established with the store
2019-12-09 17:26:42,380 - root - WARNING - Map reduce started
2019-12-09 17:26:42,380 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:28:11,718 - root - INFO - Starting Process for store
2019-12-09 17:28:11,721 - root - INFO - Client channel established with the store
2019-12-09 17:28:11,722 - root - WARNING - Map reduce started
2019-12-09 17:28:11,723 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:28:54,102 - root - INFO - Starting Process for store
2019-12-09 17:28:54,105 - root - INFO - Client channel established with the store
2019-12-09 17:28:54,106 - root - WARNING - Map reduce started
2019-12-09 17:28:54,107 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:28:54,140 - root - WARNING - SAVE INTERMEDIATE DATA: Trying to store initial data from mapper 0:mapper_input:word_count
2019-12-09 17:28:54,144 - root - WARNING - Making RPC call to store
2019-12-09 17:28:54,159 - root - WARNING - SAVE INTERMEDIATE DATA: Trying to store initial data from mapper 0:mapper_input:word_count
2019-12-09 17:28:54,160 - root - WARNING - Making RPC call to store
2019-12-09 17:28:54,164 - root - WARNING - SAVE INTERMEDIATE DATA: Trying to store initial data from mapper 0:mapper_input:word_count
2019-12-09 17:28:54,165 - root - WARNING - Making RPC call to store
2019-12-09 17:28:54,167 - root - WARNING - SAVE INTERMEDIATE DATA: Trying to store initial data from mapper 0:mapper_input:word_count
2019-12-09 17:28:54,167 - root - WARNING - Making RPC call to store
2019-12-09 17:28:54,169 - root - WARNING - SAVE INTERMEDIATE DATA: Trying to store initial data from mapper 0:mapper_input:word_count
2019-12-09 17:28:54,169 - root - WARNING - Making RPC call to store
2019-12-09 17:36:23,597 - root - INFO - Starting Process for store
2019-12-09 17:36:23,601 - root - INFO - Client channel established with the store
2019-12-09 17:36:23,602 - root - WARNING - Map reduce started
2019-12-09 17:36:23,603 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:36:23,603 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:36:23,604 - root - WARNING - Making RPC call to store
2019-12-09 17:36:23,607 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:36:23,607 - root - WARNING - Making RPC call to store
2019-12-09 17:36:23,616 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:36:23,616 - root - WARNING - Making RPC call to store
2019-12-09 17:36:23,626 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:36:23,626 - root - WARNING - Making RPC call to store
2019-12-09 17:36:23,630 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:36:23,631 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,012 - root - INFO - Starting Process for store
2019-12-09 17:42:37,015 - root - INFO - Client channel established with the store
2019-12-09 17:42:37,016 - root - WARNING - Map reduce started
2019-12-09 17:42:37,016 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:42:37,017 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:42:37,017 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,020 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:42:37,021 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,030 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:42:37,030 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,032 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:42:37,033 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,036 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:42:37,036 - root - WARNING - Making RPC call to store
2019-12-09 17:42:37,044 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,040 - root - INFO - Starting Process for store
2019-12-09 17:43:59,043 - root - INFO - Client channel established with the store
2019-12-09 17:43:59,044 - root - WARNING - Map reduce started
2019-12-09 17:43:59,045 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:43:59,046 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:43:59,046 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,049 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:43:59,050 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,056 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:43:59,057 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,059 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:43:59,059 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,062 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:43:59,062 - root - WARNING - Making RPC call to store
2019-12-09 17:43:59,064 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,958 - root - INFO - Starting Process for store
2019-12-09 17:45:44,961 - root - INFO - Client channel established with the store
2019-12-09 17:45:44,961 - root - WARNING - Map reduce started
2019-12-09 17:45:44,961 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:45:44,962 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:45:44,962 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,965 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:45:44,965 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,972 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:45:44,973 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,975 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:45:44,975 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,977 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:45:44,978 - root - WARNING - Making RPC call to store
2019-12-09 17:45:44,979 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,303 - root - INFO - Starting Process for store
2019-12-09 17:46:05,306 - root - INFO - Client channel established with the store
2019-12-09 17:46:05,306 - root - WARNING - Map reduce started
2019-12-09 17:46:05,307 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:46:05,307 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:46:05,308 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,310 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:46:05,311 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,318 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:46:05,318 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,320 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:46:05,321 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,329 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:46:05,329 - root - WARNING - Making RPC call to store
2019-12-09 17:46:05,331 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,199 - root - INFO - Starting Process for store
2019-12-09 17:46:07,202 - root - INFO - Client channel established with the store
2019-12-09 17:46:07,203 - root - WARNING - Map reduce started
2019-12-09 17:46:07,204 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:46:07,204 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:46:07,205 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,207 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:46:07,208 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,215 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:46:07,216 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,218 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:46:07,218 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,221 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:46:07,221 - root - WARNING - Making RPC call to store
2019-12-09 17:46:07,223 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,622 - root - INFO - Starting Process for store
2019-12-09 17:47:29,630 - root - INFO - Client channel established with the store
2019-12-09 17:47:29,631 - root - WARNING - Map reduce started
2019-12-09 17:47:29,632 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:47:29,632 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:47:29,633 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,635 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:47:29,635 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,637 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:47:29,637 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,639 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:47:29,639 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,641 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:47:29,641 - root - WARNING - Making RPC call to store
2019-12-09 17:47:29,653 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,299 - root - INFO - Starting Process for store
2019-12-09 17:48:10,302 - root - INFO - Client channel established with the store
2019-12-09 17:48:10,302 - root - WARNING - Map reduce started
2019-12-09 17:48:10,302 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:48:10,302 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:48:10,303 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,306 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:48:10,306 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,315 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:48:10,315 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,318 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:48:10,318 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,319 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:48:10,319 - root - WARNING - Making RPC call to store
2019-12-09 17:48:10,321 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,377 - root - INFO - Starting Process for store
2019-12-09 17:49:47,390 - root - INFO - Client channel established with the store
2019-12-09 17:49:47,392 - root - WARNING - Map reduce started
2019-12-09 17:49:47,392 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:49:47,393 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:49:47,394 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,406 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:49:47,407 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,419 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:49:47,420 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,423 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:49:47,423 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,437 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:49:47,438 - root - WARNING - Making RPC call to store
2019-12-09 17:49:47,454 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,744 - root - INFO - Starting Process for store
2019-12-09 17:50:21,747 - root - INFO - Client channel established with the store
2019-12-09 17:50:21,757 - root - WARNING - Map reduce started
2019-12-09 17:50:21,758 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:50:21,759 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:50:21,759 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,763 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:50:21,763 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,769 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:50:21,769 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,780 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:50:21,780 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,782 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:50:21,782 - root - WARNING - Making RPC call to store
2019-12-09 17:50:21,784 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,448 - root - INFO - Starting Process for store
2019-12-09 17:50:32,451 - root - INFO - Client channel established with the store
2019-12-09 17:50:32,461 - root - WARNING - Map reduce started
2019-12-09 17:50:32,461 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:50:32,462 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:50:32,463 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,466 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:50:32,466 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,473 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:50:32,474 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,475 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:50:32,475 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,477 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:50:32,478 - root - WARNING - Making RPC call to store
2019-12-09 17:50:32,490 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,798 - root - INFO - Starting Process for store
2019-12-09 17:50:51,801 - root - INFO - Client channel established with the store
2019-12-09 17:50:51,811 - root - WARNING - Map reduce started
2019-12-09 17:50:51,812 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:50:51,813 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:50:51,814 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,817 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:50:51,818 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,825 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:50:51,825 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,828 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:50:51,829 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,840 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:50:51,840 - root - WARNING - Making RPC call to store
2019-12-09 17:50:51,842 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,753 - root - INFO - Starting Process for store
2019-12-09 17:51:31,766 - root - INFO - Client channel established with the store
2019-12-09 17:51:31,766 - root - WARNING - Map reduce started
2019-12-09 17:51:31,767 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:51:31,768 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:51:31,768 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,771 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:51:31,772 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,788 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:51:31,788 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,792 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:51:31,792 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,792 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:51:31,793 - root - WARNING - Making RPC call to store
2019-12-09 17:51:31,805 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,376 - root - INFO - Starting Process for store
2019-12-09 17:53:16,388 - root - INFO - Client channel established with the store
2019-12-09 17:53:16,388 - root - WARNING - Map reduce started
2019-12-09 17:53:16,389 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:53:16,390 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:53:16,390 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,393 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:53:16,393 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,409 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:53:16,409 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,411 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:53:16,412 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,413 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:53:16,413 - root - WARNING - Making RPC call to store
2019-12-09 17:53:16,415 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,508 - root - INFO - Starting Process for store
2019-12-09 17:53:57,511 - root - INFO - Client channel established with the store
2019-12-09 17:53:57,521 - root - WARNING - Map reduce started
2019-12-09 17:53:57,521 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:53:57,522 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:53:57,523 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,526 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:53:57,526 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,544 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:53:57,544 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,545 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:53:57,545 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,547 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:53:57,547 - root - WARNING - Making RPC call to store
2019-12-09 17:53:57,549 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,808 - root - INFO - Starting Process for store
2019-12-09 17:55:12,820 - root - INFO - Client channel established with the store
2019-12-09 17:55:12,821 - root - WARNING - Map reduce started
2019-12-09 17:55:12,821 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:55:12,822 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:55:12,822 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,825 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:55:12,825 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,839 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:55:12,840 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,842 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:55:12,842 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,844 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:55:12,844 - root - WARNING - Making RPC call to store
2019-12-09 17:55:12,857 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,747 - root - INFO - Starting Process for store
2019-12-09 17:55:32,759 - root - INFO - Client channel established with the store
2019-12-09 17:55:32,759 - root - WARNING - Map reduce started
2019-12-09 17:55:32,760 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:55:32,761 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:55:32,761 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,776 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:55:32,776 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,778 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:55:32,779 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,795 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:55:32,796 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,810 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:55:32,811 - root - WARNING - Making RPC call to store
2019-12-09 17:55:32,813 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,682 - root - INFO - Starting Process for store
2019-12-09 17:56:28,694 - root - INFO - Client channel established with the store
2019-12-09 17:56:28,695 - root - WARNING - Map reduce started
2019-12-09 17:56:28,695 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:56:28,696 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:56:28,697 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,700 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:56:28,700 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,709 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:56:28,709 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,720 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:56:28,720 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,723 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:56:28,723 - root - WARNING - Making RPC call to store
2019-12-09 17:56:28,725 - root - WARNING - Making RPC call to store
2019-12-09 17:56:57,991 - root - INFO - Starting Process for store
2019-12-09 17:56:58,002 - root - INFO - Client channel established with the store
2019-12-09 17:56:58,002 - root - WARNING - Map reduce started
2019-12-09 17:56:58,003 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:56:58,004 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:56:58,004 - root - WARNING - Making RPC call to store
2019-12-09 17:56:58,007 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:56:58,007 - root - WARNING - Making RPC call to store
2019-12-09 17:56:58,015 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:56:58,015 - root - WARNING - Making RPC call to store
2019-12-09 17:56:58,017 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:56:58,017 - root - WARNING - Making RPC call to store
2019-12-09 17:56:58,029 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:56:58,029 - root - WARNING - Making RPC call to store
2019-12-09 17:56:58,031 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,475 - root - INFO - Starting Process for store
2019-12-09 17:57:59,486 - root - INFO - Client channel established with the store
2019-12-09 17:57:59,487 - root - WARNING - Map reduce started
2019-12-09 17:57:59,488 - root - WARNING - Dividing data in chunks- START
2019-12-09 17:57:59,488 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 17:57:59,489 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,492 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 17:57:59,492 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,500 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 17:57:59,500 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,502 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 17:57:59,502 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,514 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 17:57:59,514 - root - WARNING - Making RPC call to store
2019-12-09 17:57:59,517 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,067 - root - INFO - Starting Process for store
2019-12-09 18:01:32,076 - root - INFO - Client channel established with the store
2019-12-09 18:01:32,076 - root - WARNING - Map reduce started
2019-12-09 18:01:32,077 - root - WARNING - Dividing data in chunks- START
2019-12-09 18:01:32,078 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 18:01:32,078 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,087 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 18:01:32,087 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,092 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 18:01:32,092 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,094 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 18:01:32,094 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,096 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 18:01:32,096 - root - WARNING - Making RPC call to store
2019-12-09 18:01:32,097 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,001 - root - INFO - Starting Process for store
2019-12-09 18:02:26,003 - root - INFO - Client channel established with the store
2019-12-09 18:02:26,003 - root - WARNING - Map reduce started
2019-12-09 18:02:26,004 - root - WARNING - Dividing data in chunks- START
2019-12-09 18:02:26,004 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 18:02:26,005 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,008 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 18:02:26,008 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,016 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 18:02:26,017 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,019 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 18:02:26,019 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,021 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 18:02:26,021 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,022 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,024 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,026 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,036 - root - WARNING - Making RPC call to store
2019-12-09 18:02:26,038 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,959 - root - INFO - Starting Process for store
2019-12-09 18:03:25,965 - root - INFO - Client channel established with the store
2019-12-09 18:03:25,966 - root - WARNING - Map reduce started
2019-12-09 18:03:25,966 - root - WARNING - Dividing data in chunks- START
2019-12-09 18:03:25,967 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:0
2019-12-09 18:03:25,967 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,971 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:1
2019-12-09 18:03:25,971 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,983 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:2
2019-12-09 18:03:25,983 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,985 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:3
2019-12-09 18:03:25,985 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,987 - root - WARNING - SAVE INITIAL DATA: Trying to store initial data from mapper 0:mapper:word_count:4
2019-12-09 18:03:25,987 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,988 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,990 - root - WARNING - Making RPC call to store
2019-12-09 18:03:25,998 - root - WARNING - Making RPC call to store
2019-12-09 18:03:26,002 - root - WARNING - Making RPC call to store
2019-12-09 18:03:26,004 - root - WARNING - Making RPC call to store
2019-12-09 18:52:36,646 - root - INFO - Starting Worker
2019-12-09 18:52:36,653 - root - INFO - Client channel established with the store
2019-12-09 18:54:18,755 - root - INFO - Starting Worker
2019-12-09 18:54:18,764 - root - INFO - Client channel established with the store
2019-12-09 18:57:55,250 - root - INFO - Starting Worker
2019-12-09 18:57:55,258 - root - INFO - Client channel established with the store
2019-12-09 19:07:30,291 - root - INFO - Starting Worker
2019-12-09 19:07:30,294 - root - INFO - Client channel established with the store
2019-12-09 19:07:41,884 - root - INFO - Starting Worker
2019-12-09 19:07:41,891 - root - INFO - Client channel established with the store
2019-12-09 19:26:28,725 - root - INFO - Starting Worker
2019-12-09 19:26:28,728 - root - INFO - Client channel established with the store
2019-12-09 19:28:27,305 - root - INFO - Starting Worker
2019-12-09 19:28:27,312 - root - INFO - Client channel established with the store
2019-12-09 19:45:47,669 - root - INFO - Starting Worker
2019-12-09 19:45:47,672 - root - INFO - Client channel established with the store
2019-12-09 19:47:33,790 - root - INFO - Starting Worker
2019-12-09 19:47:33,793 - root - INFO - Client channel established with the store
2019-12-09 19:47:33,820 - root - CRITICAL - Failed to start the worker
2019-12-09 19:54:21,251 - root - INFO - Starting Worker
2019-12-09 19:54:21,258 - root - INFO - Client channel established with the store
2019-12-09 19:54:21,281 - root - CRITICAL - Failed to start the worker
2019-12-09 19:54:43,392 - root - INFO - Starting Worker
2019-12-09 19:54:43,398 - root - INFO - Client channel established with the store
2019-12-09 19:54:43,425 - root - CRITICAL - Failed to start the worker
2019-12-09 19:55:08,206 - root - INFO - Starting Worker
2019-12-09 19:55:08,213 - root - INFO - Client channel established with the store
2019-12-09 19:55:08,245 - root - CRITICAL - Failed to start the worker
2019-12-09 19:55:43,918 - root - INFO - Starting Worker
2019-12-09 19:55:43,921 - root - INFO - Client channel established with the store
2019-12-09 19:55:43,947 - root - CRITICAL - Failed to start the worker
2019-12-09 19:56:13,877 - root - INFO - Starting Worker
2019-12-09 19:56:13,880 - root - INFO - Client channel established with the store
2019-12-09 19:56:13,908 - root - CRITICAL - Failed to start the worker
2019-12-09 19:57:22,388 - root - INFO - Starting Worker
2019-12-09 19:57:22,396 - root - INFO - Client channel established with the store
2019-12-09 19:57:22,423 - root - CRITICAL - Failed to start the worker
2019-12-09 19:58:02,267 - root - INFO - Starting Worker
2019-12-09 19:58:02,274 - root - INFO - Client channel established with the store
2019-12-09 19:58:26,098 - root - INFO - Starting Worker
2019-12-09 19:58:26,101 - root - INFO - Client channel established with the store
2019-12-09 20:05:31,180 - root - INFO - Starting Worker
2019-12-09 20:05:31,185 - root - INFO - Client channel established with the store
2019-12-09 20:05:34,349 - root - INFO - Starting Worker
2019-12-09 20:05:34,355 - root - INFO - Client channel established with the store
2019-12-09 20:06:09,123 - root - INFO - Starting Worker
2019-12-09 20:06:09,126 - root - INFO - Client channel established with the store
2019-12-09 20:06:11,006 - root - INFO - Starting Worker
2019-12-09 20:06:11,012 - root - INFO - Client channel established with the store
2019-12-10 10:11:15,408 - root - INFO - Starting Worker
2019-12-10 10:11:15,425 - root - INFO - Client channel established with the store
2019-12-10 10:13:18,046 - root - INFO - Starting Worker
2019-12-10 10:13:18,051 - root - INFO - Client channel established with the store
2019-12-10 11:04:09,478 - grpc._server - ERROR - Exception calling application: function() argument 1 must be code, not None
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "G:\Git\worker\worker_servicer.py", line 30, in worker_reducer
    red_func = FunctionType(execCode2.co_consts[0], globals(), "foo")
TypeError: function() argument 1 must be code, not None
2019-12-10 11:13:51,254 - grpc._server - ERROR - Exception calling application: function() argument 1 must be code, not None
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "G:\Git\worker\worker_servicer.py", line 30, in worker_reducer
    red_func = FunctionType(execCode2.co_consts[0], globals(), "foo")
TypeError: function() argument 1 must be code, not None
2019-12-10 11:14:48,875 - grpc._server - ERROR - Exception calling application: function() argument 1 must be code, not None
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "G:\Git\worker\worker_servicer.py", line 30, in worker_reducer
    red_func = FunctionType(execCode2.co_consts[0], globals(), "foo")
TypeError: function() argument 1 must be code, not None
2019-12-10 11:16:16,261 - root - INFO - Starting Worker
2019-12-10 11:16:16,265 - root - INFO - Client channel established with the store
2019-12-10 11:16:32,823 - root - INFO - Starting Worker
2019-12-10 11:16:32,827 - root - INFO - Client channel established with the store
2019-12-10 11:16:32,937 - root - CRITICAL - Failed to start the worker
2019-12-10 11:16:47,402 - root - INFO - Starting Worker
2019-12-10 11:16:47,406 - root - INFO - Client channel established with the store
2019-12-10 11:16:55,234 - grpc._server - ERROR - Exception calling application: function() argument 1 must be code, not None
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "G:\Git\worker\worker_servicer.py", line 30, in worker_reducer
    red_func = FunctionType(execCode2.co_consts[0], globals(), "foo")
TypeError: function() argument 1 must be code, not None
2019-12-10 11:18:27,379 - grpc._server - ERROR - Exception calling application: function() argument 1 must be code, not None
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "G:\Git\worker\worker_servicer.py", line 30, in worker_reducer
    red_func = FunctionType(execCode2.co_consts[0], globals(), "foo")
TypeError: function() argument 1 must be code, not None
2019-12-10 13:46:42,400 - root - INFO - Starting Worker
2019-12-10 13:46:42,403 - root - INFO - Client channel established with the store
2019-12-10 13:46:54,335 - root - INFO - Starting Worker
2019-12-10 13:46:54,338 - root - INFO - Client channel established with the store
2019-12-10 16:36:42,770 - root - INFO - Starting Worker
2019-12-10 16:36:42,776 - root - INFO - Client channel established with the store
2019-12-10 16:36:57,879 - root - INFO - Starting Worker
2019-12-10 16:36:57,885 - root - INFO - Client channel established with the store
2019-12-10 16:37:08,753 - root - INFO - Starting Worker
2019-12-10 16:37:08,756 - root - INFO - Client channel established with the store
2019-12-10 16:57:11,384 - root - INFO - Starting Worker
2019-12-10 16:57:11,390 - root - INFO - Client channel established with the store
2019-12-11 00:00:53,440 - root - INFO - Starting Worker
2019-12-11 11:06:35,852 - root - INFO - Starting Worker
2019-12-11 11:06:44,191 - root - INFO - Starting Worker
2019-12-11 11:07:02,972 - root - INFO - Starting Worker
2019-12-11 11:07:08,555 - root - INFO - Starting Worker
2019-12-11 11:07:08,601 - root - CRITICAL - Failed to start the worker
2019-12-11 11:07:12,484 - root - INFO - Starting Worker
2019-12-11 11:13:01,386 - root - INFO - Starting Worker
2019-12-11 11:13:05,157 - root - INFO - Starting Worker
2019-12-11 11:14:51,664 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 56, in connect_to_store
    connect_datastore(request.address)
AttributeError: address
2019-12-11 11:17:25,849 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 56, in connect_to_store
    connect_datastore(request.address)
AttributeError: address
2019-12-11 11:18:38,268 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 56, in connect_to_store
    connect_datastore(request.address)
AttributeError: address
2019-12-11 11:19:34,593 - root - INFO - Starting Worker
2019-12-11 11:19:40,962 - root - INFO - Starting Worker
2019-12-11 11:19:50,102 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 55, in connect_to_store
    print('connect to store ', request.address)
AttributeError: address
2019-12-11 11:21:05,392 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 55, in connect_to_store
    print('connect to store ', request.address)
AttributeError: address
2019-12-11 11:21:32,663 - root - INFO - Starting Worker
2019-12-11 11:21:34,621 - root - INFO - Starting Worker
2019-12-11 11:21:36,589 - grpc._server - ERROR - Exception calling application: address
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 56, in connect_to_store
    connect_datastore(request.address)
AttributeError: address
2019-12-11 11:22:12,718 - root - INFO - Starting Worker
2019-12-11 11:22:16,910 - root - INFO - Starting Worker
2019-12-11 11:22:26,659 - root - INFO - Client channel established with the store
2019-12-11 11:22:26,659 - grpc._common - ERROR - Exception serializing message!
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_common.py", line 85, in _transform
    return transformer(message)
TypeError: descriptor 'SerializeToString' requires a 'google.protobuf.pyext._message.CMessage' object but received a 'NoneType'
2019-12-11 11:24:22,602 - root - INFO - Starting Worker
2019-12-11 11:24:24,401 - root - INFO - Starting Worker
2019-12-11 11:24:26,320 - root - INFO - Client channel established with the store
2019-12-11 11:24:26,326 - root - INFO - Client channel established with the store
2019-12-11 11:24:26,407 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:26:44,818 - root - INFO - Client channel established with the store
2019-12-11 11:26:44,820 - root - INFO - Client channel established with the store
2019-12-11 11:26:44,842 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:28:17,602 - root - INFO - Starting Worker
2019-12-11 11:28:20,101 - root - INFO - Starting Worker
2019-12-11 11:28:35,557 - root - INFO - Client channel established with the store
2019-12-11 11:28:35,560 - root - INFO - Client channel established with the store
2019-12-11 11:28:35,583 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:31:52,228 - root - INFO - Client channel established with the store
2019-12-11 11:31:52,236 - root - INFO - Client channel established with the store
2019-12-11 11:31:52,258 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:33:18,095 - root - INFO - Client channel established with the store
2019-12-11 11:33:18,102 - root - INFO - Client channel established with the store
2019-12-11 11:33:18,126 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:33:42,151 - root - INFO - Client channel established with the store
2019-12-11 11:33:42,153 - root - INFO - Client channel established with the store
2019-12-11 11:33:42,184 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:35:06,335 - root - INFO - Client channel established with the store
2019-12-11 11:35:06,343 - root - INFO - Client channel established with the store
2019-12-11 11:35:06,363 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:35:43,382 - root - INFO - Client channel established with the store
2019-12-11 11:35:43,383 - root - INFO - Client channel established with the store
2019-12-11 11:35:43,407 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:45:40,231 - root - INFO - Client channel established with the store
2019-12-11 11:45:40,240 - root - INFO - Client channel established with the store
2019-12-11 11:45:40,264 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:47:17,705 - root - INFO - Client channel established with the store
2019-12-11 11:47:17,712 - root - INFO - Client channel established with the store
2019-12-11 11:47:17,733 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:47:36,357 - root - INFO - Client channel established with the store
2019-12-11 11:47:36,359 - root - INFO - Client channel established with the store
2019-12-11 11:47:36,383 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:48:26,033 - root - INFO - Client channel established with the store
2019-12-11 11:48:26,041 - root - INFO - Client channel established with the store
2019-12-11 11:48:26,063 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:48:49,779 - root - INFO - Client channel established with the store
2019-12-11 11:48:49,786 - root - INFO - Client channel established with the store
2019-12-11 11:48:49,811 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:49:06,780 - root - INFO - Client channel established with the store
2019-12-11 11:49:06,789 - root - INFO - Client channel established with the store
2019-12-11 11:49:06,819 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:49:39,996 - root - INFO - Client channel established with the store
2019-12-11 11:49:40,005 - root - INFO - Client channel established with the store
2019-12-11 11:49:40,031 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:50:47,312 - root - INFO - Client channel established with the store
2019-12-11 11:50:47,320 - root - INFO - Client channel established with the store
2019-12-11 11:50:47,343 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:50:53,970 - root - INFO - Starting Worker
2019-12-11 11:50:57,502 - root - INFO - Starting Worker
2019-12-11 11:51:00,223 - root - INFO - Starting Worker
2019-12-11 11:51:02,553 - root - INFO - Client channel established with the store
2019-12-11 11:51:02,560 - root - INFO - Client channel established with the store
2019-12-11 11:51:02,582 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    map_func = FunctionType(execCode.co_consts[0], globals(), "foo")
NameError: name 'FunctionType' is not defined
2019-12-11 11:51:36,329 - root - INFO - Client channel established with the store
2019-12-11 11:51:36,335 - root - INFO - Client channel established with the store
2019-12-11 11:51:36,357 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:52:36,713 - root - INFO - Client channel established with the store
2019-12-11 11:52:36,715 - root - INFO - Client channel established with the store
2019-12-11 11:52:36,742 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:56:50,519 - root - INFO - Client channel established with the store
2019-12-11 11:56:50,527 - root - INFO - Client channel established with the store
2019-12-11 11:57:04,972 - root - INFO - Client channel established with the store
2019-12-11 11:57:04,981 - root - INFO - Client channel established with the store
2019-12-11 11:57:05,003 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:57:48,661 - root - INFO - Client channel established with the store
2019-12-11 11:57:48,668 - root - INFO - Client channel established with the store
2019-12-11 11:57:48,691 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:58:13,644 - root - INFO - Client channel established with the store
2019-12-11 11:58:13,652 - root - INFO - Client channel established with the store
2019-12-11 11:58:13,674 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:59:23,609 - root - INFO - Client channel established with the store
2019-12-11 11:59:23,617 - root - INFO - Client channel established with the store
2019-12-11 11:59:23,642 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 11:59:29,065 - root - INFO - Client channel established with the store
2019-12-11 11:59:29,073 - root - INFO - Client channel established with the store
2019-12-11 11:59:29,099 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:01:26,290 - root - INFO - Client channel established with the store
2019-12-11 12:01:26,296 - root - INFO - Client channel established with the store
2019-12-11 12:01:26,316 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:01:53,971 - root - INFO - Client channel established with the store
2019-12-11 12:01:53,974 - root - INFO - Client channel established with the store
2019-12-11 12:01:54,004 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:02:29,893 - root - INFO - Client channel established with the store
2019-12-11 12:02:29,894 - root - INFO - Client channel established with the store
2019-12-11 12:02:29,915 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:02:38,561 - root - INFO - Client channel established with the store
2019-12-11 12:02:38,569 - root - INFO - Client channel established with the store
2019-12-11 12:02:38,590 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:03:16,822 - root - INFO - Client channel established with the store
2019-12-11 12:03:16,831 - root - INFO - Client channel established with the store
2019-12-11 12:03:16,854 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:03:44,775 - root - INFO - Client channel established with the store
2019-12-11 12:03:44,782 - root - INFO - Client channel established with the store
2019-12-11 12:03:44,808 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:04:28,933 - root - INFO - Client channel established with the store
2019-12-11 12:04:28,939 - root - INFO - Client channel established with the store
2019-12-11 12:04:28,959 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:04:44,276 - root - INFO - Client channel established with the store
2019-12-11 12:04:44,284 - root - INFO - Client channel established with the store
2019-12-11 12:04:44,300 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:04:53,882 - root - INFO - Client channel established with the store
2019-12-11 12:04:53,884 - root - INFO - Client channel established with the store
2019-12-11 12:04:53,899 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:05:13,869 - root - INFO - Client channel established with the store
2019-12-11 12:05:13,872 - root - INFO - Client channel established with the store
2019-12-11 12:05:13,886 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:05:49,633 - root - INFO - Client channel established with the store
2019-12-11 12:05:49,640 - root - INFO - Client channel established with the store
2019-12-11 12:05:49,656 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:06:25,453 - root - INFO - Client channel established with the store
2019-12-11 12:06:25,455 - root - INFO - Client channel established with the store
2019-12-11 12:06:25,470 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:06:51,249 - root - INFO - Client channel established with the store
2019-12-11 12:06:51,256 - root - INFO - Client channel established with the store
2019-12-11 12:06:51,272 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:07:04,309 - root - INFO - Client channel established with the store
2019-12-11 12:07:04,319 - root - INFO - Client channel established with the store
2019-12-11 12:07:04,335 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:07:22,964 - root - INFO - Client channel established with the store
2019-12-11 12:07:22,966 - root - INFO - Client channel established with the store
2019-12-11 12:07:22,981 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:08:00,355 - root - INFO - Client channel established with the store
2019-12-11 12:08:00,363 - root - INFO - Client channel established with the store
2019-12-11 12:08:00,377 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:08:24,250 - root - INFO - Client channel established with the store
2019-12-11 12:08:24,257 - root - INFO - Client channel established with the store
2019-12-11 12:08:24,272 - grpc._server - ERROR - Exception calling application: name 'FunctionType' is not defined
Traceback (most recent call last):
  File "C:\Users\gurjaspal\Anaconda3\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "worker.py", line 64, in worker_map
    result = map_func(request.file_name, request.lines)
NameError: name 'FunctionType' is not defined
2019-12-11 12:09:27,912 - root - INFO - Starting Worker
2019-12-11 12:09:30,153 - root - INFO - Starting Worker
2019-12-11 12:09:39,689 - root - INFO - Client channel established with the store
2019-12-11 12:09:39,696 - root - INFO - Client channel established with the store
